leaf-ui/rappid-extensions/AnalysisModels.js

```diff
@@ -0,0 +1,88 @@
+/** Models for the Analysis Mode AnalysisInspector.js and Analysis Configuration Sidebars */
+
+/**
+ * Model for results
+ * 
+ * Attributes:
+ * {String} name
+ * {AnalysisResult} analysisResult
+ * 
+ */
+// TODO: Break AnalysisResult into params
+var ResultModel = Backbone.Model.extend({
+    idAttribute: "uid",
+    defaults: {
+        name:"Default Result",
+        analysisResult: new AnalysisResult(),
+    }
+})
+
+/**
+ * Collection that holds results for analysis configurations
+ * 
+ * {ResultModel} model
+ */
+var ResultCollection = Backbone.Collection.extend({
+    model: ResultModel,
+    url: "/results",
```

Comment:
Is the idea with collection urls to store all the results/configs in a new place on the server?

Reply:
Sort of, but to be honest I don't think it is necessary right now because that is not currently really how the communication between frontend/backend is set up. Happy to remove for now!

Reply:
Maybe just comment to clarify!